demosplan/DemosPlanCoreBundle/ResourceConfigBuilder/StatementVoteResourceConfigBuilder.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of the package demosplan.
+ *
+ * (c) 2010-present DEMOS plan GmbH, for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+namespace demosplan\DemosPlanCoreBundle\ResourceConfigBuilder;
+
+use DemosEurope\DemosplanAddon\ResourceConfigBuilder\BaseStatementVoteResourceConfigBuilder;
+use demosplan\DemosPlanCoreBundle\Entity\Statement\StatementVote;
+use EDT\DqlQuerying\Contracts\ClauseFunctionInterface;
+use EDT\JsonApi\PropertyConfig\Builder\AttributeConfigBuilderInterface;
+
+/**
+ * @property-read AttributeConfigBuilderInterface<ClauseFunctionInterface<bool>, StatementVote> $firstname
```

Comment:
Do we need this attributes? The base has these already.

Reply:
@breierDemos I've been trying to adjust the initializable behavior using the new EDT version. After several attempts that resulted in errors, I decided to use this approach to get it working. I will create another ticket related to upgrading to the newest EDT version and we can proceed to merge this PR